sdk/keyvault/keyvault-keys/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Updated the Latest service version to 7.2.
 - Added `curve` to `createKeyOptions` to be used when creating an `EC` key.
+- Deprecated the current `encrypt` and `decrypt` methods in favor of the more flexible overloads that take an `{Encrypt|Decrypt}Parameters` and allow passing in algorithm specific parameters. This enables support for the various AES algorithms used in Managed HSM. The deprecated methods continue to function and there's no timeline for their removal.
```

Comment:
Why deprecated? They should still be supported. They'll just work for RSA algorithms.